### ISSUE TITLE
aks-preview v0.5.47 breaks --load-balancer-outbound-ips arg functionality

### ISSUE THREAD

**Issue by wasker** [Azure/azure-cli#20693]
**Describe the bug**

After upgrading aks-preview to the latest version, `aks update --load-balancer-outbound-ips` is broken with error:

```
(InvalidLoadBalancerProfile) Load balancer profile must specify exactly one of ManagedOutboundIPs, OutboundIPPrefixes and OutboundIPs. Please refer to http://aka.ms/slb.
Code: InvalidLoadBalancerProfile
Message: Load balancer profile must specify exactly one of ManagedOutboundIPs, OutboundIPPrefixes and OutboundIPs. Please refer to http://aka.ms/slb.
Target: networkProfile.loadBalancerProfile
```

Side note: aka.ms/slb link points to a Microsoft internal site.

**To Reproduce**

`az aks update --resource-group rg-myrg --name aks-myaks --load-balancer-outbound-ips "/subscriptions/xxx/resourceGroups/yyy/providers/Microsoft.Network/publicIPAddresses/zzz","/subscriptions/xxx/resourceGroups/yyy/providers/Microsoft.Network/publicIPAddresses/izzz"`

or even (to just supply 1 IP addr, since error message advises that):

`az aks update --resource-group rg-myrg --name aks-myaks --load-balancer-outbound-ips "/subscriptions/xxx/resourceGroups/yyy/providers/Microsoft.Network/publicIPAddresses/zzz"`

**Expected behavior**

aks-preview v0.5.46 appropriately updates outbound IP addresses in cluster. Both commands from the repro section behave correctly.

**Environment summary**

```
Windows-10-10.0.19041-SP0
Python 3.8.9
Installer: MSI

azure-cli 2.31.0

Extensions:
aks-preview 0.5.47
azure-devops 0.22.0
front-door 1.0.16
log-analytics 0.2.1
resource-graph 2.1.0

```

**Comment by zhoxing-ms** [comment]
@FumingZhang Please help have a look at this issue~

**Comment by FumingZhang** [comment]
Sure, investigating.

**Comment by wasker** [comment]
@michaelschmit If you're using aks-preview extension v0.5.47 to do whatever you're doing, you won't be able to work this problem around unless you downgrade the extension to v0.5.46. You should be able to downgrade by using `--version` flag on the extension mgmt command of az CLI.

**Comment by FumingZhang** [comment]
Hi @wasker and @michaelschmit, sorry for the inconvenience, the issue has been fixed in v0.5.48 ([PR #4232](https://github.com/Azure/azure-cli-extensions/pull/4232)), please update to the latest version via `az extension update -n aks-preview`.

**Comment by wasker** [comment]
Thanks, it works for me now.
